Kwc/Shop/Cart/Checkout/Payment/Wirecard/Success/Component.php

```diff
@@ -13,8 +13,11 @@ public static function getSettings($param = null)
 
     public function getNameForEdit()
     {
-        return trlKwf('Shop Confirmation Text') . ' (' .$this->getData()->getSubroot()->id . ') '
-            . Kwf_Trl::getInstance()->trlStaticExecute(Kwc_Abstract::getSetting($this->getData()->parent->componentClass, 'componentName'));
+        $ret = trlKwf('Shop Confirmation Text');
```

Comment:
use data for trlKwf

Reply:
nein, indem fall nicht, weil das ist nur fürs admin und da soll er die sprache von eingeloggten benutzer nehmen